index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Oluyemi Portfolio</title>
+    <!-- CSS only -->
+    <script src="https://kit.fontawesome.com/746ba03926.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <nav id="navbar">
+            <div id="profileName">
+                <p><strong>Paul, Oluyemi</strong></p>
```

Comment:
- [ ] Why if you you put this this logo inside of an `a` tag? This is because the logo could lead to the home page from anywhere you are on the website eg: 
`<h2><a href="./index.html">Paul, Oluyemi</a></h2>`